src/main/resources/RecruitmentApp/AdminTranslations.xml

```diff
@@ -1,72 +1,71 @@
-    <number>0</number>
-    <className>XWiki.TranslationDocumentClass</className>
-    <guid>5c0d5297-ba62-405a-9e63-e07b3ce1633f</guid>
-    <property>
-      <scope>ON_DEMAND</scope>
-    </property>
-  </object>
-  <content>recruitmentapp.office.livetable.office=Office
-recruitmentapp.office.livetable._actions=Actions
-
-recruitmentapp.status.livetable.status=Status
-recruitmentapp.status.livetable._actions=Actions
-
-recruitmentapp.source.livetable.source=Source
-recruitmentapp.source.livetable._actions=Actions
-
-recruitmentapp.nextstep.livetable.step=Next Step
-recruitmentapp.nextstep.livetable._actions=Actions</content>
-</xwikidoc>
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xwikidoc version="1.3" reference="RecruitmentApp.AdminTranslations" locale="">
+  <web>RecruitmentApp</web>
+  <name>AdminTranslations</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1369914765000</creationDate>
+  <parent>WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <date>1557642498000</date>
+  <contentUpdateDate>1557642432000</contentUpdateDate>
+  <version>1.1</version>
+  <title>AdminTranslations</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content>recruitmentapp.office.livetable.office=Office
+recruitmentapp.office.livetable._actions=Actions
+
+recruitmentapp.status.livetable.status=Status
+recruitmentapp.status.livetable._actions=Actions
+
+recruitmentapp.source.livetable.source=Source
+recruitmentapp.source.livetable._actions=Actions
+
+recruitmentapp.nextstep.livetable.step=Next Step
+recruitmentapp.nextstep.livetable._actions=Actions
+
+admin.recruitment=Recruitment
+admin.recruitment.description=Allows tracking candidates and job offers</content>
+  <object>
+    <name>RecruitmentApp.AdminTranslations</name>
+    <number>0</number>
+    <className>XWiki.TranslationDocumentClass</className>
+    <guid>5c0d5297-ba62-405a-9e63-e07b3ce1633f</guid>
+    <class>
+      <name>XWiki.TranslationDocumentClass</name>
+      <customClass/>
+      <customMapping/>
+      <defaultViewSheet/>
+      <defaultEditSheet/>
+      <defaultWeb/>
+      <nameField/>
+      <validationScript/>
+      <scope>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <multiSelect>0</multiSelect>
+        <name>scope</name>
+        <number>1</number>
+        <prettyName>Scope</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators>|, </separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>GLOBAL|WIKI|USER|ON_DEMAND</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </scope>
+    </class>
+    <property>
+      <scope>GLOBAL</scope>

```

Comment:
`WIKI` would a better value rather then `GLOBAL`. I know this is not related to this issue, but maybe we can create a new issue about this, since `GLOBAL` requires "Programming rights", see https://extensions.xwiki.org/xwiki/bin/view/Extension/Localization/#HRegisterawikitranslation . A similar issue example at https://jira.xwiki.org/browse/XWIKI-15012